### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

## Related Issue

- Fixes <issue_link>

## Motivation and Context

Typos can result in tests failing to test what they intend to test. Visible typos show a lack of polish. Typos can result in poor translations. And typos can mean that when searching for an identifier, things will be missed.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...
The misspellings have been reported at https://github.com/jsoref/owncloud-web/commit/ec56747d3a825a4496614d75c04a4e63a5729def#commitcomment-56623198
The action reports that the changes in this PR would make it happy: https://github.com/jsoref/owncloud-web/commit/a0de574e81f77cbe313f3c9449cb8dab201c3d13

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/constants.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export abstract class DavPermission {
   static readonly Moveable: string = 'V'
   static readonly Updateable: string = 'NV'
   static readonly FileUpdateable: string = 'W'
-  static readonly FolderCreateable: string = 'CK'
```
Comment (jsoref) [inline packages/web-pkg/src/constants.ts:10]: This might be an API break

File: tests/report/index.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ reporter.generate({
     'web-version': pkg.version,
     platform: os.platform(),
     repository: `${repoInfo.branch}`,
-    ...(backendVersion && { [`${backendName}-verison`]: backendVersion }),
+    ...(backendVersion && { [`${backendName}-version`]: backendVersion }),
```
Comment (jsoref) [inline tests/report/index.js:50]: this shouldn't be a problem

File: changelog/0.28.0_2020-12-04/text-editor-file-path-label [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Bugfix: Don't break file/folder names in text editor
 
-The label in the text editor that displays the path of the active file was removing the first character instead of trimming leading slashes. This might have lead to situations where actual characters were removed. We fixed this by only removing leading slahes instead of blindly removing the first character.
+The label in the text editor that displays the path of the active file was removing the first character instead of trimming leading slashes. This might have lead to situations where actual characters were removed. We fixed this by only removing leading slashes instead of blindly removing the first character.
```
Comment (jsoref) [inline changelog/0.28.0_2020-12-04/text-editor-file-path-label:3]: For most of the repositories I looked at, I actively skipped the changelog/ child directories, apparently I didn't for this repository. I can trivially drop the directory if requested.

File: tests/report/index.js (hunk 2 above)
Comment (pascalwengerter) [reply tests/report/index.js:50]: Nope ;)

File: packages/web-pkg/src/constants.ts (hunk 1 above)
Comment (pascalwengerter) [reply packages/web-pkg/src/constants.ts:10]: Good call, will take a look later

File: changelog/0.28.0_2020-12-04/text-editor-file-path-label (hunk 3 above)
Comment (pascalwengerter) [reply changelog/0.28.0_2020-12-04/text-editor-file-path-label:3]: Doesn't do any harm, we're happy about the fixes/contribution!

**Review by pascalwengerter — APPROVED** [review]
Thank you so much for this! It actually detected a duplicate unit test (`packages/web-app-files/tests/unit/components/SideBar/Details/FileDetails.spec.js`) which we weren't aware of due to a typo 🥷 will squash-merge this into a meta-PR we already have so we can fix the tests & linting before bringing it to master.

Could I interest you in giving your tool a shot in https://github.com/owncloud/ocis/ and https://github.com/owncloud/owncloud-design-system?

**Comment by pascalwengerter** [comment]
@jsoref could you accept the license bot so I can merge it? :)

https://github.com/owncloud/web/pull/5815#issuecomment-921413709

**Comment by jsoref** [comment]
@pascalwengerter:
* ocis was queued, I had intended to include that in the first set, but it got missed, so it's in progress now, but i'm running out of time, so it either arrives in the next half hour or probably not for a week or two.
* license bot: I had hoped that accepting it for one owncloud project would be enough, apparently it's per project? oh well, done, I think.

**Comment by phil-davis** [comment]
> apparently it's per project?

Yes, sadly. We all have to click through the CLA acceptance the first time we contribute to each repo. (In some repos the license might be different, so there is not really a way to do an org-wide license accept)

**Comment by jsoref** [comment]
Is it possible for the bot's message to hint at this? (I've contributed to it, but haven't looked at its configuration recently)

I'm very much a drive-by contributor to things, and friction increases the likelihood of me running away/giving up.

**Comment by pascalwengerter** [comment]
> Is it possible for the bot's message to hint at this? (I've contributed to it, but haven't looked at its configuration recently)
> 
> I'm very much a drive-by contributor to things, and friction increases the likelihood of me running away/giving up.

The bot's message is auto-generated I think, but agreed it's not ideal behavior.

And yeah I figured it's worth trying to keep friction in your contribution to a minimum (hence the change to merge it into a WIP PR to resolve CI issues), grateful for your effort nonetheless!

No worries if the PR to any other repo takes another week (or more), was just keen to improve our codebases and have you test your tool some more ;)

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/19087/1/3
CI failed the lint-test pipeline. So CI master will be red now.
@pascalwengerter are you about to make another PR that corrects this?

**Comment by pascalwengerter** [comment]
> https://drone.owncloud.com/owncloud/web/19087/1/3
> CI failed the lint-test pipeline. So CI master will be red now.
> @pascalwengerter are you about to make another PR that corrects this?

@phil-davis please don't invest time here, the PR (#5792) is heavily WIP and I know exactly why the linting fails 😉 will fix this soon

**Pull request merged**